app/models/event/submission.rb

```diff
@@ -11,7 +11,8 @@ def as_json(options={})
     @assignment.
       as_json(except: [:exercise_id, :submission_id, :id, :submitter_id, :solution, :created_at, :updated_at],
               include: {
-                exercise: {only: [:id, :guide_id]},
+                guide: {only: [:id, :name]},
+                exercise: {only: [:id, :name]},
```

Comment:
@aguspina Sorry I didn't notice this before, but this is wrong. 

You should not expose guide id. It is totally internal to atheneum and has no sense outside it. 

You should expose the slug.